docs/source/features/long_context_recipe.rst

```diff
@@ -0,0 +1,117 @@
+.. _long_context_recipe:
+
+Long Context Recipe
+------------------------
+
+Long Context Model Training enhances the capability of Large Language Model to handle long context inputs. Longer sequence length is beneficial for many NLP tasks, such as document-level summarization, long document classification, and long document question answering. NeMo provides a recipe to train long context models like Llama-3, Mixtral and Nemotron.
+
+
+Access Long Context Recipe
+========================
+
+NeMo 2.0 is providing a tested recipe to train long context models. The recipe is available in the `NeMo/nemo/collections/llm/gpt/trainer` directory.
+
+Currently we are supporting the following models:
+- Llama-3
+- Mixtral
+- Nemotron
+
+Here are charts that shows the different sequence lengths that are supported by each model with different sizes:
```

Comment:
revise.

The following charts for the Llama-3, Mixtral, and Nemotron models show the different sequence lengths supported by each model at various sizes.